jmx-metrics/src/integrationTest/java/io/opentelemetry/contrib/jmxmetrics/target_systems/HbaseIntegrationTest.java

```diff
@@ -381,7 +381,7 @@ void endToEnd() {
                 metric,
                 "hbase.region_server.authentication.count",
                 "Number of client connection authentication failures/successes.",
-                "{authentication requests}",
+                "{authentication request}",
```

Comment:
[question] is there anything mentioned about spaces or the fact that is must be a single word/noun in the semantic conventions ? I think this could be something to be further discussed later but I think the fact it's about authentication is already provided by the metric name.

Reply:
I looked in the docs specifically for whitespaces on the unit labels, but I did not find anything. That's why I left it here